app/build.gradle

```diff
@@ -11,7 +11,7 @@ android {
 
     defaultConfig {
         applicationId "protect.card_locker"
-        minSdkVersion 15
+        minSdkVersion 16
```

Comment:
Why is this necessary? Can you provide a reference?

Reply:
Without this change the Travis build fails: https://travis-ci.org/brarcher/loyalty-card-locker/builds/615722153?utm_medium=notification&utm_source=github_status